packages/docs/site/docs/developers/06-apis/query-api/01-index.md

```diff
@@ -28,7 +29,7 @@ You can go ahead and try it out. The Playground will automatically install the t
 | `blueprint-url`          |                       | The URL of the Blueprint that will be used to configure this Playground instance.                                                                                                                                                                                                                                                                                                |
 | `networking`             | `yes`                 | Enables or disables the networking support for Playground. Accepts `yes` or `no`.                                                                                                                                                                                                                                                                                                |
 | `plugin`                 |                       | Installs the specified plugin. Use the plugin name from the WordPress Plugins Directory URL. For example, if the URL is `https://wordpress.org/plugins/wp-lazy-loading/`, the plugin name would be `wp-lazy-loading`. You can pre-install multiple plugins by saying `plugin=coblocks&plugin=wp-lazy-loading&…`. Installing a plugin automatically logs the user in as an admin. |
-| `theme`                  |                       | Installs the specified theme. Use the theme name from the WordPress Themes Directory URL. For example, if the URL is `https://wordpress.org/themes/disco/`, the theme name would be `disco`. Installing a theme automatically logs the user in as an admin.                                                                                                                      |
+| `theme`                  |                       | Installs the specified theme. Use the theme name from the WordPress Themes Directory URL. For example, if the URL is `https://wordpress.org/themes/disco/`, the theme name would be `disco`. Installing a theme automatically logs the user in as an admin. Multiples themes could be installed just repeating the `theme` attribute on the URL                                  |
```

Comment:
Let's add an example like with `plugin`. Also, this file might need auto-reformatting to move the last `|` further in all the rows – I think it will come up on one of the next PRs anyway.